tempodb/blocklist/poller.go

```diff
@@ -275,3 +296,34 @@ func (p *Poller) tenantIndexPollError(idx *backend.TenantIndex, err error) error
 
 	return nil
 }
+
+type backendMetaMetrics struct {
+	blockMetaTotalObjects          int
+	compactedBlockMetaTotalObjects int
+	blockMetaTotalBytes            int
```

Comment:
it's not difficult to exceed a 32 bit signed int's max value in total bytes stored. these days most things are running on 64 bit systems but if we're going to use an "int" data type we should consider the impact of it being 32 bits wide.

Maybe leave these as uint64 until they are cast to floats for the prom metric?